unicodetools/src/main/java/org/unicode/tools/emoji/Emoji.java

```diff
@@ -956,10 +938,10 @@ public static Iterable<Source> orderedEnum(Source... doFirst) {
         return ordered;
     }
 
-    public static final IndexUnicodeProperties LATEST =
+    public static final IndexUnicodeProperties PROPS =
             IndexUnicodeProperties.make(VERSION_TO_GENERATE_UNICODE);
     public static final IndexUnicodeProperties BETA =
-            IS_BETA ? IndexUnicodeProperties.make(VERSION_BETA_UNICODE) : LATEST;
+            IndexUnicodeProperties.make(VERSION_BETA_UNICODE);
```

Comment:
This is now equivalent with PROPS. Can we delete this? Any code still using it?

If you want to keep the variable, then maybe for now just assign it PROPS rather than make a new IUP object for the same version?

Reply:
I’m not prepared to fully flatten the distinction just yet, so I’m taking your suggestion of using the same IUP object.